content/applications/productivity/iot/config/flash_sdcard.rst

```diff
@@ -66,3 +66,28 @@ Lastly, click on :guilabel:`Flash` and wait for the process to finish.
 .. note::
    An alternative software for flashing the micro SD card is *Raspberry Pi Imager*. Download the
    *Raspberry Pi* software `here <https://www.raspberrypi.com/software/>`_.
+
+Handlers update
+===============
+
+There may be some instances where the drivers or interfacess need to be updated for individual
+devices. The IoT handlers (drivers and interfaces) code can be modified by syncing them with the
+corresponding server driver code.
```

Comment:
a bit confusing as well IMHO as "server driver" looks to suggest that we download from the manufacturing device as in reality it is from Odoo's configured server